setup.cfg

```diff
@@ -1,2 +1,94 @@
 [metadata]
-license_file = LICENSE
\ No newline at end of file
+name = trl
+version = 0.18.0.dev0
+description = Train transformer language models with reinforcement learning.
+long_description = file: README.md
+long_description_content_type = text/markdown
+author = Leandro von Werra
+author_email = leandro.vonwerra@gmail.com
+url = https://github.com/huggingface/trl
+keywords = transformers, huggingface, language modeling, post-training, rlhf, sft, dpo, grpo
+license_file = LICENSE
+classifiers =
+    Development Status :: 2 - Pre-Alpha
+    Intended Audience :: Developers
+    Intended Audience :: Science/Research
+    Natural Language :: English
+    Operating System :: OS Independent
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.9
+    Programming Language :: Python :: 3.10
+    Programming Language :: Python :: 3.11
+    Programming Language :: Python :: 3.12
+
+[options]
+packages = find:
+python_requires = >=3.9
```

Comment:
FYI this is set for EOL in October

Reply:
Related: python 3.13 should be soon supported: https://github.com/huggingface/trl/pull/2593